### PR TITLE
Ke/add to tokenclient

### PR DESCRIPTION
**Pull request by Ke-Jin-Laserfiche**
Export TokenClient
Add PKCEUtils

### CODE REVIEW

File: lib/OAuth/TokenClient.test.ts [hunk 1]
```diff
@@ -1,41 +1,41 @@
 import { OAuthAccessKey, testServicePrincipalKey } from '../../testHelper.js';
-import { GetAccessTokenResponse } from '../OAuth/GetAccessTokenResponse.js';
-import { TokenApiClient } from '../OAuth/TokenApiClient.js';
+import { GetAccessTokenResponse } from './GetAccessTokenResponse.js';
```
Comment (Ke-Jin-Laserfiche) [inline lib/OAuth/TokenClient.test.ts:2]: add tests for the other 2 methods

File: lib/utils/PKCEUtils.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/**
+ * Generates a random PKCE code verifier
+ * @returns PKCE code verifier
+ */
+export function generateCodeVerifier(): string {
```
Comment (Ke-Jin-Laserfiche) [inline lib/utils/PKCEUtils.ts:5]: add test

File: lib/OAuth/TokenClient.test.ts (hunk 1 above)
Comment (Ke-Jin-Laserfiche) [reply lib/OAuth/TokenClient.test.ts:2]: add tests later

File: lib/utils/PKCEUtils.ts (hunk 2 above)
Comment (Ke-Jin-Laserfiche) [reply lib/utils/PKCEUtils.ts:5]: later

**Review by paolo-arge-lf — APPROVED** [review]

**Pull request merged**